psi4/driver/procrouting/scf_proc/scf_iterator.py

```diff
@@ -292,6 +292,7 @@ def scf_iterate(self, e_conv=None, d_conv=None):
         core.timer_off("HF: Form G")
 
         incfock_performed = hasattr(self.jk(), "do_incfock_iter") and self.jk().do_incfock_iter()
+        linK_performed = hasattr(self.jk(), "do_linK") and self.jk().do_linK()
```

Comment:
I found this code hard to understand: why are these variables set here, and used much later in the code? It was not obvious from the code that the call to `form_G()` is actually `build_J` and `build_K`, which is why you check the output of the jk variables here.

I would at the very least add a comment to clarify that form_G forms the JK matrices so that it is obvious to anyone reading the code why it should be so...?